Lib/test/test_fstring.py

```diff
@@ -1358,7 +1358,6 @@ def test_conversions(self):
         self.assertAllRaise(SyntaxError, "f-string: expecting '}'",
                             ["f'{3!'",
                              "f'{3!s'",
-                             "f'{3!g'",
```

Comment:
Why was this removed?

Reply:
A side effect of this PR is that this raises an error about the conversion character, which is arguably a better error message here.

```python
❯ ./python.exe                        
Python 3.14.0a7+ (heads/tstrings:57828976882, Apr 19 2025, 17:29:57) [Clang 16.0.0 (clang-1600.0.26.6)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> f'{3!g'
  File "<python-input-0>", line 1
    f'{3!g'
         ^
SyntaxError: f-string: invalid conversion character 'g': expected 's', 'r', or 'a'
```